osbuild/monitor.py

```diff
@@ -90,6 +91,11 @@ class LogMonitor(BaseMonitor):
     the log will get written to. If `fd`  is a `TTY`, escape
     sequences will be used to highlight sections of the log.
     """
+    def result(self, result):
+        # Add the assembler/stage duration to the output.
+        self.duration = int(time.time()) - self.timer_start
+        self.out.write(f"\n⏱  Duration: {self.duration}s\n")
+
```

Comment:
```suggestion
    def result(self, result):
        duration = int(time.time() - self.timer_start)
        self.out.write(f"\n⏱  Duration: {duration}s\n")

```

Again, the comment sounds redundant to me, but that might just be me? But you can drop `self.` from the `duration` variable, since local scope is sufficient. And then I would apply the `int()` conversion to the calculated result, based on my comment below.

We might argue that we should introduce the `module()` function to `BaseMonitor` as well, so we don't accidentally add another module-specific callback which `LogMonitor` does not implement, and then suddenly we forget to call `module()`. But... this is outside this scope. Similarly, the duration calculation could then be done in `BaseMonitor()`. Just thinking out loud here. I guess we should tackle this when adding `JsonMonitor`.